app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -400,6 +409,8 @@ private void processChooserResponse(Intent intent) {
             Toast.makeText(FormEntryActivity.this,
                     Localization.get("form.attachment.invalid"),
                     Toast.LENGTH_LONG).show();
+        } else if (FileUtil.isFileTooLargeToUpload(getContentResolver(), media)) {
```

Comment:
this seems to be check on original file again instead of the scaled image

Reply:
yeah this is for audio and video. We only compress images, so the check is on the original file.

Reply:
Makes sense